packages/react-dom/src/client/ReactDOMRoot.js

```diff
@@ -27,6 +26,7 @@ export type RootOptions = {
     onDeleted?: (suspenseNode: Comment) => void,
     ...
```

Comment:
I feel like the option should probably go into this object that has the hydration related stuff. It doesn't really do anything if you're not hydrating and you shouldn't feel obligated to put all your mutableSources here if they're not used for hydration.

Reply:
Sure.